src/app/components/bibliography/bibliography-item/bibliography-item.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { BibliographicCitation } from 'src/app/services/xml-parsers/bibliography-parser.service';
+
+@Component({
+  selector: 'evt-bibliography-item',
+  templateUrl: './bibliography-item.component.html',
+  styleUrls: ['./bibliography-item.component.scss'],
+})
+export class BibliographyItemComponent implements OnInit {
+  @Input() biblField: BibliographicCitation;
+  templateBiblCitData;
+  ngOnInit() {
+    this.templateBiblCitData = Object.values(this.biblField);
```

Comment:
You can remove this if you use `keyvalue` pipe in template

Reply:
This is what I consequently did to overcome the above compilation command.